Documentation/ceph-cluster-crd.md

```diff
@@ -150,6 +150,8 @@ The `mon` pod does not allow `Pod` affinity or anti-affinity.
 This is because of the mons having built-in anti-affinity with each other through the operator. The operator chooses which nodes are to run a mon on. Each mon is then tied to a node with a node selector using a hostname.
 See the [mon design doc](https://github.com/rook/rook/blob/master/design/mon-health.md) for more details on the mon failover design.
 
+Rook creates a Job to detect the installed Ceph version. This Job only observes the `all` placement.
```

Comment:
```
The Rook Ceph operator creates a Job called `rook-ceph-detect-version` to detect the full Ceph version used by the given `cephVersion.image`. The placement from `all` section is used for the Job.
```
Sounds better from my perspective. What do you think?

Reply:
I think clarifying `kubernetes job` may be a good addition as well.